internal/c/libqb/build.mk

```diff
@@ -0,0 +1,10 @@
+
+QB_LIBQB_SETUP_OBJS := $(PATH_INTERNAL_C)/libqb.o
+
```

Comment:
I realized after looking it over that this file isn't actually getting used. I will be using this file in upcoming changes to work on splitting apart `libqb` but I'll remove it from this PR.